Fix bug in spork which prevents it from working with rspec3
--- a/lib/spork/test_framework/rspec.rb	2014-10-20 00:30:34.865710236 -0500
+++ b/lib/spork/test_framework/rspec.rb	2014-10-20 00:33:07.058583740 -0500
@@ -1,3 +1,7 @@
+# patched as shown in https://github.com/manafire/spork/commit/38c79dcedb246daacbadb9f18d09f50cc837de51?diff=unified
+
+require 'rspec/core/version'
+
 class Spork::TestFramework::RSpec < Spork::TestFramework
   DEFAULT_PORT = 8989
   HELPER_FILE = File.join(Dir.pwd, "spec/spec_helper.rb")
@@ -7,11 +11,19 @@
       ::Spec::Runner::CommandLine.run(
         ::Spec::Runner::OptionParser.parse(argv, stderr, stdout)
       )
+    elsif rspec3?
+      options = ::RSpec::Core::ConfigurationOptions.new(argv)
+      ::RSpec::Core::Runner.new(options).run(stderr, stdout)
     else
       ::RSpec::Core::CommandLine.new(argv).run(stderr, stdout)
     end
   end

+  def rspec3?
+    return false if !defined?(::RSpec::Core::Version::STRING)
+    ::RSpec::Core::Version::STRING =~ /^3\./
+  end
+
   def rspec1?
     defined?(Spec) && !defined?(RSpec)
   end
